### PR TITLE
DE-1125 Disable syslog outputs

### PR DESCRIPTION
**Pull request by pchojnacki**
- change name of parameter events endpoint
- refactor outputs configurations

### CODE REVIEW

**Comment by WikiaTech** [comment]
SonarQube analysis reported 3 issues:
* MAJOR 2 major
* MINOR 1 minor

Watch the comments in this conversation to review them.
Note: the following issues could not be reported as comments because they are located on lines that are not displayed in this pull request:
* MINOR [Remove this unused import 'io.dropwizard.validation.ValidationMethod'.](https://github.com/Wikia/pandora/blob/16253d0831111ae7b25454a70933ad62152e4269/service/clickstream/src/main/java/com/wikia/service/clickstream/ClickstreamConfiguration.java#L4) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AUselessImportCheck)
* MAJOR [Move this file to a named package.](https://github.com/Wikia/pandora/blob/16253d0831111ae7b25454a70933ad62152e4269/service/clickstream/src/test/java/com/wikia/service/clickstream/ClickstreamConfigurationTest.java) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1220)
* MAJOR [Move this file to a named package.](https://github.com/Wikia/pandora/blob/16253d0831111ae7b25454a70933ad62152e4269/service/clickstream/src/test/java/com/wikia/service/clickstream/sender/SenderTest.java) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1220)

**Comment by sqreek** [comment]
LGTM

File: service/clickstream/src/main/java/com/wikia/service/clickstream/resources/EventResource.java [hunk 1]
```diff
@@ -31,9 +31,9 @@
   @POST
   @Timed
   @Consumes(MediaType.APPLICATION_JSON)
-  @Path("/{appName}")
+  @Path("/{loggerName}")
   @CacheControl(maxAge = 0, maxAgeUnit = TimeUnit.SECONDS)
-  public Response postBatch(Events events, @PathParam("appName") String appName) {
+  public Response postBatch(Events events, @PathParam("loggerName") String appName) {
```
Comment (sqreek) [inline service/clickstream/src/main/java/com/wikia/service/clickstream/resources/EventResource.java]: is appName should be loggerName now?

**Comment by WikiaTech** [comment]
SonarQube analysis reported 1 issue:
* MINOR 1 minor

Watch the comments in this conversation to review them.
Note: the following issues could not be reported as comments because they are located on lines that are not displayed in this pull request:
* MINOR [Remove this unused import 'io.dropwizard.validation.ValidationMethod'.](https://github.com/Wikia/pandora/blob/aec027f9b9bf8c0a6111a88eaeda18a9706bbb03/service/clickstream/src/main/java/com/wikia/service/clickstream/ClickstreamConfiguration.java#L4) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AUselessImportCheck)

**Pull request merged**